generators/aws-sdk-rbs-generator/Rakefile

```diff
@@ -21,6 +21,14 @@ task :write do |task, args|
   service_args.each { |name| services[name].write_files(dir:) }
 end
 
-task :default do
-  Rake::Task['write'].invoke("S3", "SQS", "KMS", "EC2")
+task :update_src do
+  cd "_src" do
+    sh "git pull origin version-3"
+  end
 end
+
+task :generate_all do
+  Rake::Task['write'].invoke("S3", "SQS", "KMS", "EC2", "SSM")
```

Comment:
Ah this looks like what I was expecting but failed to find - I think this should be included in the documentation you've added (i.e. that only services in this "array" are covered, and that it's here that new services should be added)

Reply:
@G-Rath Thank you for reviewing.
How about with the additions to the [generators/aws-sdk-rbs-generator/README.md](https://github.com/ruby/gem_rbs_collection/pull/191/files#diff-7820d191b29cfc265b704758b3dca4fb22e907a62dd9775fa033cc81ebe2de68)?

Reply:
I was thinking something more explicit i.e. "Add the client service to the list in the `generate_all` task".

(tbh its probably worth pulling that out into an array constant since it could grow quite large over time, and then you could say something like "Add the missing service to the `AWS_SERVICES` array and run `bundle exec rake`")

Reply:
@G-Rath Looks Good, I've added that sentence. Thanks!
This generator is still in a phase of gradual expansion as we listen to feedback, but eventually we intend to eliminate the need to add to the list.